changelog.md

```diff
@@ -1,5 +1,80 @@
+- Add: [Script] Saving/loading ScriptList (#13556)
+- Add: Press Ctrl to build diagonal canals in game mode (#13432)
+- Add: Sandbox settings to Sandbox Options window (#13268)
+- Add: Setting to allow placing houses manually in-game (#13266)
+- Add: [Script] Event for when a company's president name changes (#13208)
+- Add: Ability to toggle visibility of station signs by facility (#13207)
+- Add: [Script] ScriptEventCompanyRename (#12878)
+- Add: Ukrainian Hryvnia currency (#12877)
+- Change: Make tree placement at world generation look more organic (#13515)
+- Change: [MacOS] Put the icon in a rounded rectangle (#13446)
+- Change: [Script] GetWaypointID to return the StationID of any waypoint (#13407)
+- Change: Draw company manager face jacket after collar (#13390)
+- Change: Don't distinguish between bus and truck stops when removing them (#13384)
+- Change: [Script] Rename BridgeID to BridgeType in the script API (#13352)
+- Change: Add fonts document to help window (#13305)
+- Change: Log changes to sandbox settings (#13267)
+- Change: When player joins network company, use its name instead of number in chat (#13263)
+- Change: [Win32] Draw window title bar according to current Windows light/dark theme (#13196)
+- Change: Restore wider spacers in main toolbars (#12039)
+- Fix #13842: Close industry production graph if industry is removed (#13890)
+- Fix: Autoreplace rail/road list only listed buildable types (#13887)
+- Fix: [NewGRF] Display an error, if NewGRF reference out-of-bounds string parameters in gender/plural choices (#13881)
+- Fix #13849: Settings in old saves could be overridden by defaults (#13874)
+- Fix #13562: Removed cost estimation message from money cheat (#13857)
+- Fix: [NewGRF] Plurals and genders did not work in strings with cases or substrings (#13853, #13852)
+- Fix: [NewGRF] String parameter stack and case selection were not processed for control code 0x81 (#13851)
+- Fix #13839: Incorrect colour of first company legend in smallmap window (#13841)
+- Fix: i circumflex width in TrueType small font (#13836)
+- Fix: Don't show owner of non-existent road (#13824)
+- Fix: Error message window timeout doesn't match setting (#13812)
+- Fix #13795: Crash in vehicle list of 32-bit platforms (#13796)
+- Fix: [Script] Company rename event sometimes had the wrong name (#13794)
+- Fix: Improve manager face randomisation (#13776)
+- Fix #13740: [Script] Handle implicit orders for jump orders (#13753)
+- Fix #13749: Default service intervals were not updated when changing timekeeping unit (#13751)
+- Fix #13725: Use proper query strings for changing timetable values (#13737)
+- Fix #11226: Don't draw story page elements that won't be visible (#13736)
+- Fix: More AI than max_no_competitors could start with competitors_interval=0 (#13670)
+- Fix: League table window ignored the minimal size in its widget description (#13629)
+- Fix: Incorrect snow density when making rocks snowy (#13626)
```

Comment:
Not player-facing until snowy rocks were added, which was after beta1.

```suggestion
```

Reply:
How I read the PR is that it happened before 'snowy' rocks. If a rocky tile were made snowy, it would become fully snowed and then it would thaw a bit. In effect causing a white 'flash' for the rocky tiles.

Reply:
Yes, it was visible before snowy rocks, only with a NewGRF that introduces varying snow line.